README.md

```diff
@@ -916,27 +947,27 @@ which would use the same general rules as `|>`.
 `+>` would be a **prefix operator** that **creates a new function**,
 which in turn **binds its argument(s)** to topic references.
 **Non-unary functions** would be created
-by including topic references with **numbers** (`%0`, `%1`, `%2`, etc.) or `...`.
-`%0` (equivalent to plain `%`) would be bound to the **zeroth argument**,
-`%1` would be bound to the next argument, and so on.
-`%...` would be bound to an array of **rest arguments**.
+by including topic references with **numbers** (`0`, `1`, `2`, etc.) or `...`.
```

Comment:
Not sure if this one was unintentional or not

```suggestion
by including topic references with **numbers** (`@0`, `@1`, `@2`, etc.) or `...`.
```

I could also see it being

```suggestion
by including topic references with **numbers** (`@0`, `@1`, `@2`, etc.) or `@...`.
```

to match the lines below